qiskit_optimization/problems/linear_expression.py

```diff
@@ -172,3 +173,41 @@ def evaluate_gradient(self, x: Union[ndarray, List, Dict[Union[int, str], float]
 
         # extract the coefficients as array and return it
         return self.to_array()
+
+    def _bounds(self) -> Tuple[float, float]:
```

Comment:
This could easily be public method and made a property so you can just do `l_b,  u_b = le.bounds`.

In place of have other methods/properties you could use a `namedtuple` so you could access the values by sat 'lower' and 'upper' e.g `le.bounds.lower` (the namedtuple just makes it a little more readable than doing `le.bounds[0]`

As to the larger question of whether this is needed or not I have no insight to that to offer.

Reply:
Thank you for your idea. Namedtuple or dataclass sounds better.